### PR TITLE
READme for example amf and smf packages

### PR DESCRIPTION
**Pull request by Smitholi67**

### CODE REVIEW

File: pkg-example-amf-bp/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# pkg-amf
+
+## Description
+example pkg for amf blueprint
+
+## Usage
```
Comment (johnbelamaric) [inline pkg-example-amf-bp/README.md:6]: I would just remove the Usage sections, and expand on the description to describe what the package is for. "Used to deploy a free5gc AMF..." etc.

Eventually, in here we would describe if this package accepts any inputs, config injection etc.

Same comment for the SMF

Comment (henderiw) [reply pkg-example-amf-bp/README.md]: agree I would keep the part that is related to how to use the package.

maybe make a reference to the process this package should go through to reach deployment. Not all details but the high level parts

File: pkg-example-amf-bp/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# pkg-amf
+
+## Description
+Example blueprint package for free5gc AMF network function
+
+## The blueprint package is modeled by multiple parties working together including
```
Comment (johnbelamaric) [inline pkg-example-amf-bp/README.md:6]: These shouldn't all be headers

**Review by johnbelamaric — CHANGES_REQUESTED** [review]
\#\# means "second level header" in Markdown. See https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

**Comment by johnbelamaric** [comment]
Ok, this is a start. I think we'll want to get more into how these fit into the entire flow, but we can add that later.

**Pull request merged**